x-pack/plugins/enterprise_search/server/routes/app_search/engines.test.ts

```diff
@@ -107,4 +107,30 @@ describe('engine routes', () => {
       });
     });
   });
+
+  describe('GET /api/app_search/engines/{name}', () => {
```

Comment:
I thought this made sense in the same file as the collective/plural `/api/app_search/engines`, but lmk if you disagree!